packages/customize-widgets/README.md

```diff
@@ -0,0 +1,6 @@
+- Package name
```

Comment:
Let's fill this out very very briefly. Just a one sentence explanation about how this package is for the UI that appears in the customizer is enough for now.